### DISCUSSION TITLE
downloading auto ortho

### DISCUSSION THREAD

**Discussion by dsec1** [kubilus1/autoortho#210] [Q&A]
How long should it take to extract the downloads to the custom scenery folder in the auto ortho downloads? I downloaded quite quickly, but the extraction for north america is going to take about 4 hours...is this normal.

the downloader detected a split zip and now it's cleaning up parts for each of the 29 tiles and it is taking forever...don't think this is right

**Comment by kubilus1** [comment]
Probably due to Windows Defender!  I was shocked when testing how terrible
it makes the performance.

**Reply by dsec1** [reply]
I finally made it through the extraction but encountered an error: It seems I'm missing terrain overlay files in the z_na_08 folder. Can I manually add these to the folder if they are available somewhere to cut and paste? Or, is there a diagnostic tool available that can tell me if any additional files are missing? I used Ortho for xplane 11 and would like to see it up and running for xplane 12

**Comment by dpdurst** [comment]
The entire process for me took about 40 mins to an hour total to download and unzip. I've done it twice now and never ran into an issue where it missed anything.

**Comment by hersh81** [comment]
I've tried installing north america multiple times and download goes quick and is roughly 25gb, but extracting takes 4 or 5 hours and it's only through 1/2 the extraction and already used 350GB of space in the custom scenery folder.  Is this normal?  if so there is no mention of this anywhere.  How much space do you need for north america?

**Reply by dpdurst** [reply]
Entire time to install for me was 40 min to an hour with a 1GB connection. Total size I believe was about 33GB of space

**Reply by hersh81** [reply]
Interesting,  i've tried it about 4 different times and currently in process again.  It downloads all the files total of 25gb.  But the extraction process takes hours and like I said i'm only on part 08 of 29 with 331gb already in custom scenery
Any suggestions?  i've followed the document and also watched 2 different install tutorial videos

**Reply by dpdurst** [reply]
Well something is not correct, not sure how you are getting 350GB out of those files. the yautoortho_overlays folder is 3 GB and the z_autoortho folder is about 29.1 when all is said and done.

Lets start with your autoortho log file first

**Reply by dsec1** [reply]
I'm trying again - extraction takes longest - hours. Downloader downloaded y_na_overlays.zip00 y_na_overlays.zip01 and y_na_overlays.zip03 before extraction. And then it cleans up parts for each na folder which takes about 20 minutes to clean up.

**Reply by hersh81** [reply]
So I just figured out what my problem was, I had moved some storage to an external drive and it was formatted to Fat32 which I didn't realize. And this is where I had my scenery folder.  I cleared out some room on my NVME drive and it worked as it should.  With it being a Fat32 partition, the size on disk quadrupled from what it should have been.  Thank you for your assistance though.  
I managed to get it installed and working today.

**Reply by dsec1** [reply]
Good to hear hersh...I am still "cleaning up parts for C:\Users\danie\.autoortho-data\downloads\z_na_20.zip" i.e extracting the download to the scenery folder in xplane (I assume). It's 21:41 eastern, I started the with the install from scratch at around 17:30. I'm going to end up going to bed rather than waiting for the other ... z_na_21.zip just finised...eight folders to clean up (whatever that means) and when I check in the morning I will find an error message that something crashed at the end. That has happened twice now. I'm reluctant to keep doing it because it takes so long ... won't try much longera. I'll let you know how I make out

**Reply by dsec1** [reply]
on z_na_26.zip it's 23:16 est about six hours now. 3 more to go

**Comment by dsec1** [comment]
didn't work
![Screenshot (2)](https://github.com/kubilus1/autoortho/assets/136533890/926d9ca4-1dd8-4e82-b9e8-91c4bc65af4d)

**Reply by dpdurst** [reply]
So do you not have an NTFS volume? log is saying you don't.

**Reply by dsec1** [reply]
now it renders x plane useless untill i remove all the orth files i just spent 7 hours downloading and extracting unless i can fix this

**Reply by dsec1** [reply]
where would i find it?

**Reply by dpdurst** [reply]
You can open a command prompt and type diskpart, then list volumes and look for E drive. Should show to the right if its NTFS or FAT32 etc. I just did mine shown below see the column that says NTFS

![volume](https://github.com/kubilus1/autoortho/assets/8261471/55cb013b-9eab-4caa-9819-b4da9048cd39)

**Reply by dsec1** [reply]
says it's exFAT

**Reply by dpdurst** [reply]
Well there is your issue and why its not working, it needs to be NTFS for it to install to that volume

**Reply by dpdurst** [reply]
Highlighted the error in your log

![error](https://github.com/kubilus1/autoortho/assets/8261471/a32948f8-f752-4cbd-99d8-7ac269493af8)

**Reply by dsec1** [reply]
gotcha, so if i stored x plane on C drive, which I see is NTFS, that should work?

**Reply by dsec1** [reply]
![Screenshot (3)](https://github.com/kubilus1/autoortho/assets/136533890/021f36c0-36d3-47d8-871f-4c771cdfaa1c)

**Reply by dpdurst** [reply]
As long as you have the space requirements I would say yes it should work. Not sure what you have on E drive itself if not a lot maybe then just format that drive for NTFS (plenty of space) and use it.

**Reply by dsec1** [reply]
I wonder if i should delete all of the ortho files first, then move to x plane folder to C drive and reinstall ortho? or could I just move everything over as is?

**Reply by dpdurst** [reply]
Depends again on your setup and what you mean by ortho files and moving them. Let me explain best I can

-- You have a path you can set for 5.0 on where you download the ortho packages to. These normally get deleted once the install is complete unless you change the setting in the ortho config file to save the downloaded files. If you have done this then those downloaded files will remain in that folder saving time if you delete everything and install again.

-- If you don't do the above save of the downloaded files, then of course you will re-download them all again on the next setup of AO.

-- Moving XP over to C drive is a matter of just copying that entire folder for the sim. Your shortcut may not work because its pointed to the old location but of course you can make a new shortcut of the XP executable from the new location so good to go. You could try to move the ortho files as well and then just change the path of where they are located and see if it picks up or not. If not then you start over.

-- Me personally I'd probably just move the XP folder over and then redo the installation of the AO program so you know for 100% its right at that time.

hopefully this all made sense..

**Reply by dsec1** [reply]
lol, how would i reformat the e drive for NTFS?

**Reply by dpdurst** [reply]
Fair warning, I'll explain below, if your not familiar with this then maybe that's not an option for you

With that being said, if you running windows you can open up windows explorer then right click on the drive and in the context menu that opens up you will see the command format. Choosing that you have an option to select NTFS (normal or fast format) and then the Start button to do its thing. It will delete everything on that drive permanently and the drive will be like brand new

**Reply by dsec1** [reply]
Well, trying again...I reformated the E drive from exFAT to NTFS. Success! Reinstalled xp12. Success! Reinstalled AO. Downloaded fine. Extraction taking forever.  It is currently 11:30 hrs est and I'm Extracting C:\Users\danie\.autoortho-data\downloads\z_na_03.zip   I'm anticipating the extraction will be completed around 20:00 hrs est - seems like a loooong time.

**Comment by kubilus1** [comment]
Is windows defender live protection or whatever it's called in?  That will
slow things down a ton.   Look in you task manager and see if what's taking
resources.

**Reply by dsec1** [reply]
k, ty - i will....

**Reply by dsec1** [reply]
not sure how to read this - is it the NVIDIA thing slowing everything down?
![Screenshot (4)](https://github.com/kubilus1/autoortho/assets/136533890/d7efa547-aed0-42e0-b7aa-84fc5676164f)

**Reply by dsec1** [reply]
that animalware service executable is the antivirus

**Reply by dsec1** [reply]
are there things i can shut down for now to speed things up?

**Reply by dsec1** [reply]
i shut down windows defender and Steam client - doesn't seem to be speeding anything up

**Reply by dsec1** [reply]
Could it be that my E drive is plugged into a USB? I wonder if it would have been faster just using the C drive

**Reply by dsec1** [reply]
![Screenshot (5)](https://github.com/kubilus1/autoortho/assets/136533890/e7812b65-90ff-49ef-a5db-be909cdf8b16)

**Reply by dsec1** [reply]
guess I just gotta wait out the extraction

**Reply by dsec1** [reply]
![Screenshot (6)](https://github.com/kubilus1/autoortho/assets/136533890/7d5f9862-109f-41b8-b57b-12986ff2ffaf)

**Reply by dsec1** [reply]
I may be able to get this up and running by tonight lol

**Reply by dsec1** [reply]
haven't figured out how to stop that antimalwar executable...

**Reply by dsec1** [reply]
![Screenshot (7)](https://github.com/kubilus1/autoortho/assets/136533890/0ecb6037-81b9-4e9b-85df-fb94f800034c)

**Reply by dsec1** [reply]
I'm thinking if i had installed this on the SSD rather than USB drive it may have been a lot faster. I just hope I'm not waiting in vain

**Reply by dpdurst** [reply]
Mine is installed on an m.2 drive and it takes about 40min to an hour total to install it. So SSD would be way faster than the USB connection

I doubt your AV or Defender is causing any issue at all -> USB

**Comment by kubilus1** [comment]
Yeah installing over USB will be slow.  Looks like you are getting only
70KB/s.

**Comment by kubilus1** [comment]
I'd be surprised if this works from USB drive as well.  It's possible, but
not something I've ever tried or tested.

**Reply by dsec1** [reply]
Well...My next move is to move stuff from C drive to the E drive to give myself enough room, then install on C...

**Comment by dsec1** [comment]
Thanks, I'll let you know

**Comment by dsec1** [comment]
Cleaning up parts now - going much quicker
